bin/rpm-s3

```diff
@@ -107,15 +107,16 @@ class FileGrabber(object):
 
 
 def getclient(base, region):
+    host_url = "s3.amazonaws.com" if region == "us-east-1" else "s3-{}.amazonaws.com".format(region)
     if os.getenv('AWS_ACCESS_KEY'):
         return boto.connect_s3(
             os.getenv('AWS_ACCESS_KEY'),
```

Comment:
@crohr Why not use the AWS environment variable names (i.e. `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`)?